sync_check_terraform/main.tf

```diff
@@ -0,0 +1,178 @@
+terraform {
+  backend "s3" {
+    # Note: This is the bucket for the internal terraform state. This bucket is
+    # completely independent from the bucket that contains snapshots.
+    bucket = "forest-iac"
+    # This key uniquely identifies the service. To create a new service (instead
+    # of modifying this one), use a new key. Unfortunately, variables may not be
+    # used here.
+    key = "sync_check.tfstate"
+
+    # This value is completely unused by DO but _must_ be a known AWS region.
+    region = "us-west-1"
+    # The S3 region is determined by the endpoint. fra1 = Frankfurt.
+    # This region does not have to be shared by the droplet.
+    endpoint = "https://fra1.digitaloceanspaces.com"
+
+    # Credentially can be validated through the Security Token Service (STS).
+    # Unfortunately, DigitalOcean does not support STS so we have to skip the
+    # validation.
+    skip_credentials_validation = "true"
+  }
+}
+
+locals {
+  name                  = "forest-sync-check" # droplet name
+  size                  = "so-2vcpu-16gb"     # droplet size
+  image                 = "fedora-36-x64"
+  slack_channel         = "#forest-notifications" # slack channel for notifications
```

Comment:
Can we keep those variables in the same way as in the daily snapshot? I don't know which one is the "preferred" way of doing things in Terraform, but it'd be nice to have a common service structure in the repo so it is easy to switch between them cognitively. 

https://github.com/ChainSafe/forest-iac/blob/main/daily_snapshot_terraform/main.tf

Reply:
done